sonic-net/sonic-utilities
# Kdump enhancement for remote ssh, cli subcommands added.

#### What I did
Worked on the enhancement of kdump utility for sonic. Added two new subcommands along with two options i.e
1- sudo config kdump remote ssh --conn-str ___ --key-path ___
2- sudo config kdump remote disable
#### How I did it
Made changes to 3 files in total,
1- config/kdump (added two new subcommands and options for modentry to config_db)
2- show/kdump (cli command to print running config)
3- scripts/sonic-kdump-config (scripts or ready methods for usage upon load and unload of kdump module)
#### How to verify it
upon loading this utilities' build to the running sonic image we can try and test these commands for verifying it's behavior as per the requirements.
